src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const listItems = document.querySelectorAll('li');
```

Comment:
```suggestion
const listItems = [...document.querySelectorAll('li')];
```
можна далі без Array.from використовувати)